FEWD/Week02/lesson03/homework/relaxr_starter_code/css/style.css

```diff
@@ -64,59 +65,119 @@ body {
   width: 100%;
   margin: auto;
   height: 500px;
-  display: block;
 }
 
 header {
+  height: 500px;
+}
 
+h1 {
+  color: white;
+  font-weight: bolder;
+  font-size: 50px;
+  padding: 25px;
+  margin-left: 20px;
 }
 
-h3 {
-  background-color: #f0efef
+h2 {
+  color: white;
+  font-size: 40px;
+  font-weight: 400;
+  padding: 75px;
+  margin-left: 250px;
```

Comment:
So my main thought here is that you're having trouble centering things. Hopefully Tuesday's review helped with that. For example, to centre this `h2`

```css
margin: 0;
text-align: center;
```

`h2` is a block element which extends the full width of the page. It's usually not a brilliant idea to add margins to block elements that are 100% wide. The idea is that margins should create separation between items that are not 100% wide on the page.

As `h2` is a block element and it contains text, `text-align: center;` does the trick, while leaving the `h2` at 100%